README.md

```diff
@@ -1 +1,30 @@
 # EDPN
+Elite Dangerous Pilot Network front-end
+
+## Installation
```

Comment:
Should we leave a note about a minimum Node version to be installed. I ran this setup initially on an old workspace running v10 of node and the installing had issues with vite. May be we recommend running Node LTS?

Reply:
Of course! I'll add this info on the Readme file to recommend Node LTS

Reply:
Looked into vite docs https://vitejs.dev/guide/, looks like "Vite requires [Node.js](https://nodejs.org/en/) version 14.18+, 16+."